projects/imagemagick/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "https://www.imagemagick.org"
+primary_contact: "dirk@git.imagemagick.org"
+ccs:
+  - paul.l.kehrer@gmail.com
+  - alex.gaynor@gmail.com
```

Comment:
MemorySanitizer seems to work well with these fuzz target, can you please add the following piece of project.yaml:

```
sanitizers:
  - address
  - memory
  - undefined
```
that will enable all the configurations we use